class.jetpack-cli.php

```diff
@@ -1476,6 +1476,107 @@ private function partner_provision_error( $error ) {
 		) ) );
 		exit( 1 );
 	}
+
```

Comment:
This would go out in production, but only really be helpful in prod. Would it make sense to have this in an included-in-dev only file? `class.jetpack-cli-dev.php` or something that we exclude from production/built versions?

Reply:
Given our time constraints right now, let's focus on getting this out of the door and consider refactor files later.